client/e2e/pages/authenticated/routes/page.spec.ts

```diff
@@ -50,119 +58,121 @@ function buildAccessLists(currentRole: UserRole): AppRoute[] {
 
 // 🏷 Annotate test suite as serial
 test.describe.configure({ mode: "serial" });
-// ➰ Loop through the entries of UserRole enum
-for (let [role, value] of Object.entries(UserRole)) {
-  role = "E2E_" + role;
-  const storageState = process.env[role + "_STORAGE"] as string;
-  test.describe(`Test Route Access for ${value}`, () => {
-    // 👤 Run test as this role
-    test.use({ storageState: storageState });
-    test("Test Navigate to Routes", async ({ page }) => {
-      // 🚨 Build routes allowed list for this user role
-      const accessLists = buildAccessLists(value);
+test.describe("Test Route Access", () => {
+  // ➰ Loop through the entries of UserRole enum
```

Comment:
Are these changes related to fixing the flakiness, or separate thing?

Reply:
@BCerki there was 1 flaky test in this file for so I reconfigured the loop to get better details on the role and the route in hopes to help troubleshoot.